cmd/kosli/reportEvidenceArtifactGeneric_test.go

```diff
@@ -121,15 +121,15 @@ func (suite *ArtifactEvidenceGenericCommandTestSuite) TestArtifactEvidenceGeneri
 			          --build-url example.com --compliant --description "some description"
 					  --evidence-paths non-existing %s`,
 				suite.artifactFingerprint, evidenceName, suite.flowName, suite.defaultKosliArguments),
-			golden: "Error: stat non-existing: no such file or directory\n",
+			goldenRegex: "Error: stat non-existing: no such file or directory\n",
 		},
 		{
 			name: "report Generic test evidence fails if --name is missing",
 			cmd: fmt.Sprintf(`report evidence artifact generic --fingerprint %s --flow %s
 			          --build-url example.com %s`,
 				suite.artifactFingerprint, suite.flowName, suite.defaultKosliArguments),
-			wantError: true,
-			golden:    "Error: required flag(s) \"name\" not set\n",
+			wantError:   true,
+			goldenRegex: "Error: required flag\\(s\\) \"name\" not set\n",
```

Comment:
Any reason for changing to reg ex? Besides that we have to escape special characters?

Reply:
@ToreMerkely as all the commands are deprecated every test failed because the actual output always started with a deprecation warning.

It felt like noise in the tests to add that everywhere so I switched them to regex rather than exact match.